Cargo.toml

```diff
@@ -38,6 +38,7 @@ num-modular = { version = "0.6", features = ["num-bigint", "num-integer", "num-t
 proptest = "1"
 rand_core = { version = "0.6", features = ["std"] }
 rand_chacha = "0.3"
+crypto_bigint_05 = {package = "crypto-bigint", version = "0.5.5"}
```

Comment:
Would you mind removing this dependency and the tests that depend on it, leaving only tests against the current codebase with fixed vectors?

Reply:
I mean, this PR is just a throw-away thing to tinker with a tricky bug and this was convenient for that purpose. If there's anything we salvage here long term it's perhaps the test vectors, but perhaps we can put those in your other PR?